iOS/ExampleSuperApp.xcodeproj/project.pbxproj

```diff
@@ -652,6 +575,79 @@
 			defaultConfigurationName = Release;
 		};
 /* End XCConfigurationList section */
+
+/* Begin XCRemoteSwiftPackageReference section */
+		2B1EDFC22AA11282006D6673 /* XCRemoteSwiftPackageReference "GXUIApplication" */ = {
+			isa = XCRemoteSwiftPackageReference;
+			repositoryURL = "https://github.com/GeneXus-SwiftPackages/GXUIApplication.git";
+			requirement = {
+				kind = upToNextMinorVersion;
+				minimumVersion = 1.1.2;
+			};
+		};
+		2B1EDFC52AA112E6006D6673 /* XCRemoteSwiftPackageReference "GXDataLayerRemote" */ = {
+			isa = XCRemoteSwiftPackageReference;
+			repositoryURL = "file:///Users/jechague/Developer/GitHub/iOSFlexibleClient/Build/.gxbuild/SwiftPackages/GXDataLayerRemote";
```

Comment:
This should point to the remote public repo instead.

Reply:
Fixed in https://github.com/genexus-books/gx-super-app/pull/12/commits/fbd416303adc3f50e64a449111b2de712332307b